### PR TITLE
style(ui): refresh icons

### PR DESCRIPTION
**Pull request by rudolfs**
Screenshot 2020-08-24 at 17 42 21

preview

- [x] update all icons according to [new style guide](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8C%B1%F0%9F%86%99-Radicle-Upstream?node-id=3554%3A17)
- [x] convert all icons to a unified size - 24px x 24px
- [x] make naming consistent across style guide in Figma and code
- [x] list all icons in design system guide programmatically (so we don't have to update it whenever we add/remove or change icons)
- [x] make icons copyable in the design system guide

Closes: https://github.com/radicle-dev/radicle-upstream/issues/797.

### CODE REVIEW

File: ui/DesignSystem/Component/EmptyState.svelte [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
   import { createEventDispatcher } from "svelte";
-  import { Button, Icon } from "../Primitive";
+  import { Button } from "../Primitive";
+  import Illustration from "./EmptyState/Illustration.svelte";
 
   const dispatch = createEventDispatcher();
 
   export let style = null;
-  export let icon = "plant";
+  export let illustration = "plant"; // eyes | plant | telescope | tent
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/Component/EmptyState.svelte:9]: Yeah, I wanted to use TS, but couldn't figure out how to get it to work with Svelte. We should probably do this in a separate PR and across the whole codebase.

File: ui/Screen/SendFunds.svelte [hunk 2]
```diff
@@ -216,7 +216,7 @@
           validation={$amountValidation}>
           <div slot="left" style="display: flex;">
             <Icon.Currency
-              size="big"
+              variant="rad-big"
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/Screen/SendFunds.svelte]: Moved all the icon variants into separate components.

**Review by brandonhaslegs — APPROVED** [review]

File: ui/DesignSystem/Component/EmptyState.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarahscott) [reply ui/DesignSystem/Component/EmptyState.svelte:9]: You can use typescript in svelte so long as the typescript is in a `.ts` file - e.g.:

`validation.ts`
```typescript
export enum ValidationStatus {
  NotStarted = "NOT_STARTED",
  Loading = "LOADING",
  Error = "ERROR",
  Success = "SUCCESS",
}
```

`Text.svelte`
```typescript
<input
    data-cy={dataCy}
    class:invalid={validation && validation.status === ValidationStatus.Error}
    class:left-item={showLeftItem}
    {placeholder}
    bind:value
    {disabled}
    on:change
    on:input />
```

[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/Component/EmptyState.svelte:9]: Would we then have a `ComponentTypes.ts` file and put all the variants in there?

[the remark this answers is not in the document]
Comment (xla) [reply ui/DesignSystem/Component/EmptyState.svelte:9]: It would probably be `illustration.ts` which exports an `enum Variant`.

**Review by xla — APPROVED** [review]
Besides the improvements to variant usage, this looks good.

🚊 🛰 💜 🐢

**Review by MeBrei — CHANGES_REQUESTED** [review]
Looks good to me!
I would also extract the variants as @sarahscott suggested.

**Review by MeBrei — APPROVED** [review]
🎉

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]
- [x] `<Icon.Info />` should be `<Icon.InfoCircle />`
- [x] `<Icon.InfoNoCircle />` should be `<Icon.Info />`
- [ ] `<Icon.CurrencyDollarLarge />` should be USD instead of Dollar? "Dollar" is used in many currencies (Canadian Dollar for example). I think the currency in general should use the three letter acronym. So USD, EUR, and RAD.
- [ ] Remove `Large` from all icons. Only have `Small` in the case where there is a second size.
- [ ] Change `<Icon.Roadmap />` to `<Icon.Road />` because it's more neutral.
- [ ] Remove all with `NoCircle`. There should only be Circle as an add on if there's a circle around the icon.
- [ ] We were missing the `LockSmall` icon.
- [ ] Ellipsis is spelled with an i, not an e.

File: ui/DesignSystem/Component/EmptyState.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/Component/EmptyState.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/806/commits/abe4961f630f2a9716f9168888153e362742af8d

File: ui/DesignSystem/Component/EmptyState/Illustration.svelte [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
   export let style = null;
-  export let variant = "plant"; // eyes | plant | sight | tent
+  export let variant = "plant"; // eyes | plant | telescope | tent
 </script>
 
-<IconWrapper on:click {dataCy} {style} width="48" height="48">
+<svg
+  data-cy={dataCy}
+  on:click
+  style={`flex-shrink: 0;${style ? style : ''}`}
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply ui/DesignSystem/Component/EmptyState/Illustration.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/806/commits/260d4fc919f582be1237854deb781a815f92e56f

**Review by brandonhaslegs — APPROVED** [review]

**Review by MeBrei — APPROVED** [review]

**Pull request merged**